OpenSim/Simulation/RegisterTypes_osimSimulation.cpp

```diff
@@ -162,9 +162,9 @@ OSIMSIMULATION_API void RegisterTypes_osimSimulation()
     Object::registerType( PathPointSet() );
     Object::registerType( ConditionalPathPoint() );
     Object::registerType( MovingPathPoint() );
-    Object::registerType( BaseAppearance());
+    Object::registerType( SurfaceAppearance());
+    Object::registerType( CurveAppearance());
     Object::registerType( Appearance());
```

Comment:
`SurfaceAppearance` and `CurveAppearance` probably should not be exposed as individual objects since Appearance (which contains both)  is the only way to access/modify.